### PR TITLE
Add scraper for Louisiana Ct. of Appeals 1st Circuit

### PR DESCRIPTION
**Pull request by mmantel**
* Added scraper for the [1st Circuit of the Louisiana Court of Appeals](https://www.la-fcca.org/).
* Tweaked `string_utils.harmonize` to recognize upper-case "VS" as the court uses.

### CODE REVIEW

**Comment by mlissner** [comment]
This looks pretty great as a first PR, thank you! If you have any suggestions for docs that would have made your life easier, we'd love to have those tweaks too.

I'm usually the person that reviews these PRs, but I'm in Chile this week at a conference. @arderyp, is the other big maintainer of Juriscraper, so he might jump in, but it's Thanksgiving week, so my guess is it might take a minute to get this reviewed.

What drew you to this project and to this scraper in particular?

**Comment by mmantel** [comment]
Cool, no rush.

> What drew you to this project and to this scraper in particular?

Nothing in particular, it's just an area I'm interested in, and I noticed that the coverage of state appeals courts is pretty good, but Louisiana was missing.

**Comment by mmantel** [comment]
I resolved merge conflicts that had arisen since this PR was opened.

I also updated test cases for a couple courts that were affected by replacing the upper-case "VS" or "Vs" with "v.", which seems to be an improvement. Happy to split that change out to a separate PR if this diff is too large.

**Comment by flooie** [comment]
Thanks @mmantel - this will be reviewed this week. Thank you for your patience and for re-upping this.

File: juriscraper/opinions/united_states/state/lactapp_1.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        yield from range(1, last_page + 1)
+
+    def _get_last_page_number(self):
+        html = self._get_html_tree_by_url(self._base_url, {})
+
+        # Text is something like "Results: 1 - 50 of 16,753"
+        text = html.cssselect("#frmPagingopinionUpper td")[0].text_content()
+        count_str = re.search(" of ([0-9,]+)", text).group(1)
+        count = int(count_str.replace(",", ""))
+        return int(math.ceil(count / float(self._page_size)))
```
Comment (flooie) [inline juriscraper/opinions/united_states/state/lactapp_1.py:74]: This seems unnecessary.  the last page is in html.  I just get nervous when I see the need to do math (unless its necessary).  

I think you can get the last page with something like this 

```        
pg_str = self.html.xpath("//a[@title='last']/@onclick")[0]

pg = re.findall(r"(\d+)", pg_str)[-1]
```

File: juriscraper/opinions/united_states/state/lactapp_1.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+class Site(OpinionSiteLinear):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.court_id = self.__module__
+        self._page_size = 50
+        self._base_url = (
+            "https://www.la-fcca.org/opiniongrid/opinionpub.php?opinionpage_size=%d"
+            % self._page_size
+        )
```
Comment (flooie) [inline juriscraper/opinions/united_states/state/lactapp_1.py:27]: I think we are trying to use f-strings when we can.

File: juriscraper/opinions/united_states/state/lactapp_1.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            )
+
+    def _parse_docket_numbers(self, row):
+        # Handle cases such as:
+        # "2018CA1765<br>Rehearing<br>Application"
+        #     => "2018CA1765"
+        # "2018CA1742<br>Consolidated With<br>2018CA1743"
+        #     => "2018CA1742, 2018CA1743"
+        text = get_row_column_text(row, 2)
+        case_numbers = re.findall("[0-9]{4}[A-Z]{2,}[0-9]{4,}", text)
```
Comment (flooie) [inline juriscraper/opinions/united_states/state/lactapp_1.py:55]: This regex could be tightened up a bit.  It looks like a bit broad.

**Comment by flooie** [comment]
@mmantel this looks great.  Minus a few small tweaks, I think this is pretty close.

**Comment by mmantel** [comment]
Thanks @flooie, updated per your comments.

**Comment by flooie** [comment]
This is great.  @mmantel Thanks for sticking this one out.

**Comment by mlissner** [comment]
Woah, thank you all!

**Pull request merged**